ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/port-channel-interfaces.j2

```diff
@@ -80,6 +80,21 @@ interface {{ port_channel_interface }}
 {%         if port_channel_interfaces[port_channel_interface].rt is arista.avd.defined %}
       route-target import {{ port_channel_interfaces[port_channel_interface].rt }}
 {%         endif %}
+{%         if port_channel_interfaces[port_channel_interface].redundancy is arista.avd.defined %}
+      redundancy {{ port_channel_interfaces[port_channel_interface].redundancy }}
+{%         endif %}
+{%         if port_channel_interfaces[port_channel_interface].designated_forwarder_election is arista.avd.defined and port_channel_interfaces[port_channel_interface].designated_forwarder_election.algorithm is arista.avd.defined %}
+{%             set designated_forwarder_cli = "designated-forwarder election algorithm " %}
+{%             if port_channel_interfaces[port_channel_interface].designated_forwarder_election.algorithm is arista.avd.defined("modulus") %}
+{%                set designated_forwarder_cli = designated_forwarder_cli ~ port_channel_interfaces[port_channel_interface].designated_forwarder_election.algorithm %}
+{%             elif port_channel_interfaces[port_channel_interface].designated_forwarder_election.algorithm is arista.avd.defined("preference") %}
+{%                set designated_forwarder_cli = designated_forwarder_cli ~ port_channel_interfaces[port_channel_interface].designated_forwarder_election.algorithm ~ " " ~ port_channel_interfaces[port_channel_interface].designated_forwarder_election.preference_value %}
+{%                if port_channel_interfaces[port_channel_interface].designated_forwarder_election.dont_preempt is arista.avd.defined(true) %}
+{%                   set designated_forwarder_cli = designated_forwarder_cli ~ " dont-preempt" %}
+{%                endif%}
+{%             endif %}
+      {{ designated_forwarder_cli | indent(3, false) }}
```

Comment:
The `indent` filter is only for multiline CLI output. And even if you had multiline, the value `3` does not match the spaces you have (6). I suggest to remove the `indent` filter here.